bucket/composer.json

```diff
@@ -1,39 +1,41 @@
 {
+    "version": "1.9.0",
     "homepage": "https://getcomposer.org/",
     "description": "Dependency Manager for PHP",
     "license": "MIT",
-    "version": "1.9.0",
     "url": "https://getcomposer.org/download/1.9.0/composer.phar",
+    "hash": "c9dff69d092bdec14dee64df6677e7430163509798895fbd54891c166c5c0875",
+    "pre_install": [
+        "@(",
+        "    'if($args.length -eq 1 -and ($args -eq \"selfupdate\" -or $args -eq \"self-update\")) { & scoop update composer }'",
+        "    'else { & php (join-path $psscriptroot \"composer.phar\") @args }'",
+        ") | Add-Content -Path \"$dir\\composer.ps1\"",
+        "$och = \"$env:APPDATA\\Composer\"",
```

Comment:
What does `och` mean?

Reply:
I don't know... I remove it.

Reply:
Was introduced with https://github.com/lukesampson/scoop/pull/1510